mailserver/mailserver.go

```diff
@@ -53,9 +55,17 @@ var (
 	archivedErrorsCounter  = metrics.NewRegisteredCounter("mailserver/archiveErrors", nil)
 )
 
+type DB interface {
+	Close() error
+	Write(*leveldb.Batch, *opt.WriteOptions) error
+	Put([]byte, []byte, *opt.WriteOptions) error
+	Get([]byte, *opt.ReadOptions) ([]byte, error)
```

Comment:
`Get` method is not used on the current code, so, you don't need it as part of this interface.

Reply:
@adriacidre it's used in `mailserver_test` when testing the `Archive` function